weldx/asdf/file.py

```diff
@@ -139,6 +143,10 @@ class WeldxFile(_ProtectedViewDict):
         When `False`, when reading files, attempt to memory map (memmap) underlying data
         arrays when possible. This avoids blowing the memory when working with very
         large datasets.
+    array_inline_threshold :
+        arrays below this threshold will be serialized as string, if larger as binary
+        block. Note that this not affect arrays, which are being shared across several
+        objects in the same file.
```

Comment:
```suggestion
        block. Note that this does not affect arrays, which are being shared across several
        objects in the same file.
```
Same as above (already mentioned by @CagtayFabry )